src/index.js

```diff
@@ -1,10 +1,44 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+
 import './style.css';
```

Comment:
Let's create the minimal styling, our structure should be like this 👇 
![image](https://user-images.githubusercontent.com/31897434/129821881-7d511245-4dd1-4615-98cb-b2b55cbd51d5.png)


In another word; let's try to clone the example given on the project page above.